forum/profiles/models.py

```diff
@@ -38,6 +39,7 @@ def __str__(self):
             f")"
         )
 
+
 class InvestorProfile(models.Model):
     user = models.OneToOneField(User, on_delete=models.CASCADE, related_name="investor_profile", db_index=True)
     country = models.CharField(max_length=50)
```

Comment:
Nice code, impressive work. However, I have a suggestion about `country` fields for profiles... 
What do you think about switching it to `CountryField` from django-countries? It’s got built-in validation, uses ISO standards, and works nicely with serializers and forms, so it might save us from having to deal with typos or extra validation down the line. 

https://pypi.org/project/django-countries/#django-rest-framework - here's some documentation in case you decide to implement it.

Reply:
That's nice idea. But I would suggest to implement this as a separate task because here I was just using existing models that were created within separate ticket.